ding/worker/collector/interaction_serial_evaluator.py

```diff
@@ -187,6 +187,8 @@ def eval(
             - stop_flag (:obj:`bool`): Whether this training program can be ended.
```

Comment:
you can add a subclass of `InteractionSerialEvaluator` to implement the parts of code related to trading, and push into `dizoo/gym_anytrading/worker`